src/components/menu.js

```diff
@@ -3,6 +3,19 @@ import classnames from "classnames";
 
 import styles from "./menu.scss";
 
+const registerToggleActive = (add, toggleActive, parentIframeId) => {
+  add
+    ? window.addEventListener("click", toggleActive)
+    : window.removeEventListener("click", toggleActive);
+
+  const parentIframe = top.document.getElementById(parentIframeId);
+
+  if (parentIframe) {
```

Comment:
OK, discussed this with Yarin.
I understand the complexity so I'd like to suggest something a bit more generic - 

Lets have a prop (for my lack of naming skills) called parents.
This prop will get an array of elements that will have the event listeners added and removed.
By default, we append window to that array so that we always listen to window.

This will help us avoid things like parentIframe which is very implementation specific.

Sounds reasonable?

Reply:
Yup, done tested and works